cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowImpl1.qll

```diff
@@ -379,6 +378,8 @@ class PathNode instanceof I::PathNode {
   final predicate isSinkGroup(string group) { super.isSinkGroup(group) }
 }
 
+module PathGraph = I::PathGraph;
```

Comment:
Shouldn't we instead be redefining the `nodes`, `edges`, and `subpaths` predicates in terms of the `PathNode` type defined above?

Reply:
We could. Can you think of a case where it matters?